pumps/stdout.go

```diff
@@ -0,0 +1,117 @@
+package pumps
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/TykTechnologies/logrus"
+	"github.com/TykTechnologies/tyk-pump/analytics"
+	"github.com/mitchellh/mapstructure"
+)
+
+var (
+	stdOutPrefix     = "stdout-pump"
+	stdOutDefaultENV = PUMPS_ENV_PREFIX + "_STDOUT" + PUMPS_ENV_META_PREFIX
+)
+
+type StdOutPump struct {
+	CommonPumpConfig
+	conf *StdOutConf
+}
+
+type StdOutConf struct {
+	EnvPrefix    string `mapstructure:"meta_env_prefix"`
+	Format       string `mapstructure:"format"`
+	LogFieldName string `mapstructure:"log_field_name"`
+}
+
+func (s *StdOutPump) GetName() string {
+	return "Stdout Pump"
+}
+
+func (s *StdOutPump) GetEnvPrefix() string {
+	return s.conf.EnvPrefix
+}
+
+func (s *StdOutPump) New() Pump {
+	newPump := StdOutPump{}
+	return &newPump
+}
+
+func (s *StdOutPump) Init(config interface{}) error {
+
+	s.log = log.WithField("prefix", stdOutPrefix)
+
+	s.conf = &StdOutConf{}
+	err := mapstructure.Decode(config, &s.conf)
+
+	if err != nil {
+		s.log.Fatal("Failed to decode configuration: ", err)
+	}
+
+	processPumpEnvVars(s, s.log, s.conf, stdOutDefaultENV)
+
+	if s.conf.LogFieldName == "" {
+		s.conf.LogFieldName = "tyk-analytics-record"
+	}
+
+	s.log.Info(s.GetName() + " Initialized")
+
+	return nil
+
+}
+
+/**
+** Write the actual Data to Stdout Here
+ */
+func (s *StdOutPump) WriteData(ctx context.Context, data []interface{}) error {
+	s.log.Debug("Attempting to write ", len(data), " records...")
+
+	//Data is all the analytics being written
+	for _, v := range data {
+
+		select {
+		case <-ctx.Done():
+			return nil
+		default:
+			// Decode the raw analytics into Form
+			decoded := v.(analytics.AnalyticsRecord)
+			message := Json{
+				"timestamp":       decoded.TimeStamp,
+				"method":          decoded.Method,
+				"path":            decoded.Path,
+				"raw_path":        decoded.RawPath,
+				"response_code":   decoded.ResponseCode,
+				"alias":           decoded.Alias,
+				"api_key":         decoded.APIKey,
+				"api_version":     decoded.APIVersion,
+				"api_name":        decoded.APIName,
+				"api_id":          decoded.APIID,
+				"org_id":          decoded.OrgID,
+				"oauth_id":        decoded.OauthID,
+				"raw_request":     decoded.RawRequest,
+				"request_time_ms": decoded.RequestTime,
+				"raw_response":    decoded.RawResponse,
+				"ip_address":      decoded.IPAddress,
+				"host":            decoded.Host,
+				"content_length":  decoded.ContentLength,
+				"user_agent":      decoded.UserAgent,
+			}
```

Comment:
I can see a few more fields missing we should definitely add - basically lets get the whole analytics record in there! 

Things like tags are super important

https://github.com/TykTechnologies/tyk-pump/blob/c9d9b92ecea143423e40acdfa5e2d3d71b5bb63d/analytics/analytics.go#L28

Reply:
```json 
{"level":"info","msg":"","time":"0001-01-01T00:00:00Z","tyk-analytics-record":{"method":"PUT","host":"","path":"/widget/unconvergent","raw_path":"/widget/unconvergent","content_length":609,"user_agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/602.4.8 (KHTML, like Gecko) Version/10.0.3 Safari/602.4.8","day":5,"month":5,"year":2021,"hour":22,"response_code":200,"api_key":"588b52eab275ff0001cc7476ff2be18c970949ac81a30582abf9c445","timestamp":"2021-05-05T22:42:07.133794-03:00","api_version":"Default","api_name":"Foo Bar","api_id":"a47f35f411f14c837b99e99519e2432c","org_id":"588b52eab275ff0001cc7476","oauth_id":"","request_time":3,"raw_request":"Qk9EWSBEQVRB","raw_response":"UkVTUE9OU0UgREFUQQ==","ip_address":"118.93.55.103","geo":{"country":{"iso_code":""},"city":{"geoname_id":0,"names":null},"location":{"latitude":0,"longitude":0,"time_zone":""}},"network_stats":{"open_connections":0,"closed_connections":0,"bytes_in":0,"bytes_out":0},"latency":{"total":0,"upstream":0},"tags":["orgid-588b52eab275ff0001cc7476","apiid-a47f35f411f14c837b99e99519e2432c"],"alias":"","track_path":true,"expireAt":"2022-05-27T22:42:07.625944-03:00"}}
```

Something like that would do the trick? I think I'm not missing any field 👀